xbmc/cores/VideoPlayer/VideoPlayerRadioRDS.cpp

```diff
@@ -969,19 +968,6 @@ unsigned int CDVDRadioRDSData::DecodeTA_TP(uint8_t *msgElement)
   return 4;
 }
 
-unsigned int CDVDRadioRDSData::DecodeMS(uint8_t *msgElement)
```

Comment:
I understand not complete why it becomes removed? Can you describe it.

If it becomes removed are also other parts not needed anymore. The value `m_MS_SpeechActive` and also the parts from `CPVRRadioRDSInfoTag` need then removed.

To leave the parts on `CPVRRadioRDSInfoTag` and `CGUIInfoManager` make remove of them no sense.